Automattic/jetpack
# Upsale SEO tools feature on Simple sites

Fixes https://github.com/Automattic/jetpack/issues/34582

## Proposed changes:

This is the followup for https://github.com/Automattic/jetpack/pull/34546. Instead of hiding the feature entirely on Simple sites, we want to show Upsale message instead.

<img width="279" alt="Screenshot 2023-12-19 at 13 52 58" src="https://github.com/Automattic/jetpack/assets/60262784/316e630c-a370-4abc-8ced-3f757924ee85">


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


Tracking event added `jetpack_seo_tools_upsell_click`

## Testing instructions:






* Checkout branch locally and build `jetpack build plugins/jetpack`
* Point your Simple and Atomic (site with business plan) to your sandbox and use `rsync` command
* When creating new post, in your Jetpack sidebar you should see an upsell message only on Simple sites (lower than business)
* Check buttons behavior
* Verify that on Atomic and Self hosted message doesn't appear.
